client/src/controllers/UnsavedController.ts

```diff
@@ -360,8 +361,11 @@ export class UnsavedController extends Controller<HTMLFormElement> {
         if (!(form instanceof HTMLFormElement)) return;
 
         [
-          ['data-w-unsaved-confirmation-value', confirmationMessage || ' '],
-          ['data-w-unsaved-force-value', String(alwaysDirty || false)],
+          [
+            'data-w-unsaved-confirmation-value',
+            `${!!confirmationMessage || true}`,
+          ],
```

Comment:
Could you double check how I handled the coercion of `confirmationMessage` for the legacy `enableDirtyFormCheck` here?

It previously has `confirmationMessage || ' '`. Note the non-empty string there, it contains a space. Was that intentional? For this reason, I'm changing this to `|| true`.

However, this means you can't use `enableDirtyFormCheck` in a way that results in `data-w-unsaved-confirmation-value="false"`. This was already the case before this PR btw because `someFalsy || ' '` would be `' '`, which is truthy.